src/App.js

```diff
@@ -103,63 +104,179 @@ class App extends Component<Props, State> {
       return <LoginRoute login={this._dataStore.login} />;
     }
 
+    const navLinkStyle = {
+      color: '#fff',
+      filter: 'grayscale(100%)',
+      transition: 'filter 500ms ease-in-out',
+      cursor: 'pointer',
+    };
+
+    const navActiveLinkStyle = {
+      color: '#1da1f2',
+      filter: 'grayscale(0%)',
+    };
+
+    const navLinkSvgStyle = {
+      width: '2rem',
+      height: '2rem',
+      fill: 'currentColor',
+    };
+
+    const createLinkStyle = {
+      position: 'absolute',
+      bottom: '1rem',
+      right: '1rem',
+      width: '3rem',
+      height: '3rem',
+      padding: '0.75rem',
+      borderRadius: '3rem',
+      background: '#eceff1',
+    };
+
     return (
       <Router>
-        <div className="app">
-          <header className="header">
-            <div className="header-title-row">
-              <div className="header-user-avatar">
+        <div
+          className="app"
+          {...css({
```

Comment:
This is sort of subjective, but as I mentioned [in my comment on the issue](https://github.com/bvaughn/personal-logger/issues/6#issuecomment-354054826), I think [using Glamor's `createElement` is cleaner](https://github.com/threepointone/glamor/blob/master/docs/createElement.md).